src/Javis.jl

```diff
@@ -67,32 +71,158 @@ end
 abstract type Transition end
 abstract type InternalTransition end
 
-abstract type ActionType end
+abstract type AbstractAction end
+
+"""
+    SubAction <: AbstractAction
+
+Defines a subaction with relative frames and a function.
+
```

Comment:
I see that you explain how the subaction works in the following function definitions but it is actually not explicitly stated "what" a subaction is and why you'd want to use them. Could you add a bit of exposition about what a SubAction actually is? Thanks!

Reply:
I hope it's clearer now. In general the example in the constructor should be quite simple to understand, I hope.